flake-modules/dev/devshell.nix

```diff
@@ -13,6 +13,7 @@
     }:
     lib.optionalAttrs (inputs.devshell ? flakeModule) {
       devshells.default = {
+        packages = [ pkgs.python3 ];
```

Comment:
Any reason why you're adding python to the devshell instead of packaging your script as a derivation (like we normally do)?